.circleci/config.yml

```diff
@@ -18,8 +18,21 @@ test-defaults: &test-defaults
 
     - run: |
         apt-get update
+        echo "keyboard-configuration keyboard-configuration/layoutcode string us" | debconf-set-selections
         apt-get install -y python3 cmake build-essential openjdk-9-jre-headless $TEST_DEPENDENCY
+        if [ x"$DISPLAY" != x ]; then
```

Comment:
Please add a comment here, maybe something like "start an openbox session so the browser can render (if we move to headless mode, this may not be needed eventually)" - ?